packages/frontend/src/pages/bridges-tvl/BridgesTvlView.tsx

```diff
@@ -26,13 +26,14 @@ export interface BridgesTvlViewEntry {
   slug: string
   warning?: string
   isArchived?: boolean
+  isUpcoming?: boolean
   isVerified?: boolean
-  tvl: string
-  tvlBreakdown: TVLBreakdownProps
-  oneDayChange: string
-  sevenDayChange: string
-  bridgesMarketShare: string
-  combinedMarketShare: string
+  tvl?: string
+  tvlBreakdown?: TVLBreakdownProps
+  oneDayChange?: string
+  sevenDayChange?: string
+  bridgesMarketShare?: string
+  combinedMarketShare?: string
```

Comment:
What do you think about making some sort of type logic here? So those would be undefined for Upcoming Rollup, but required for existing.

Reply:
I would stick to my solution just for now, in the future we can do the refactor